packages/pyright-internal/src/analyzer/pythonPathUtils.ts

```diff
@@ -29,8 +29,6 @@ interface PythonPathResult {
     prefix: string;
 }
 
-const cachedSearchPaths = new Map<string, PythonPathResult>();
```

Comment:
I think we're caching enough in the import resolver that the removal of this cache won't matter. Out of curiosity, did you do any benchmarking to confirm this?

Reply:
I haven't; I can try and clone your team's repo to test.

Note that the import resolver is already caching this call explicitly (was before per venv though that code got modified), so I wouldn't expect it to have changed.

Reply:
I can't find your repo anymore, but on typeshed there is no difference in the numbers I get when using `--stats`.

Reply:
I'm not that concerned about it. I can test it after you land this change.